bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # So, we want to find the rototrans (aka homogeneous transform) of ulna and the humerus from local to global coordinates.
</pre><p>Don't mix the commentaries. Put it in code cells only if it's necessary.</p><p>print the matrices</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>